src/components/Toolbar.vue

```diff
@@ -1,15 +1,38 @@
 <template id="toolbar">
 
     <div class="toolbar">
-        <router-link v-bind:to="'/'"><i class="fa fa-keyboard-o" aria-hidden="true"></i></router-link>
-        <router-link v-bind:to="'/scan'"><i class="fa fa-camera" aria-hidden="true"></i></router-link>
-        <router-link v-bind:to="'/upload'"><i class="fa fa-upload" aria-hidden="true"></i></router-link>
+        <transition name="fade" mode="out-in">
+            <div :key="voucherCount" v-if="vouchersAdded" class="count"><router-link v-bind:to="'/payment'"><strong>{{ voucherCount }}</strong> voucher<span v-if="voucherCount > 1">s</span> waiting</router-link></div>
+        </transition>
+        <div class="input-icons">
+            <router-link v-bind:to="'/'"><i class="fa fa-keyboard-o" aria-hidden="true"></i></router-link>
+            <router-link v-bind:to="'/scan'"><i class="fa fa-camera" aria-hidden="true"></i></router-link>
+            <router-link v-bind:to="'/upload'"><i class="fa fa-upload" aria-hidden="true"></i></router-link>
```

Comment:
scan and upload are blue, tap is grey. Muted or pastel colours (like grey) often denote "deactivated"; consider 
 swapping, so that the active tool is in the richer colour.